test/e2e/minimatch/minimatch_test.go

```diff
@@ -15,17 +15,13 @@
 package minimatch
```

Comment:
the refactor here is un-related to this PR? Intentionally here?

Reply:
I placed these helper functions in the wrong file in the last PR :( Wanna revert them back in one PR since these moves does not hurt readability that much.